src/app/src/test/kotlin/org/vaccineimpact/orderlyweb/tests/integration_tests/tests/web/RunReportPageTests.kt

```diff
@@ -120,4 +127,60 @@ class RunReportPageTests : IntegrationTest()
                 ReportWithDate("minimal", repo.getLatestVersion("minimal").date)
         )
     }
+
+    @Test
+    fun `can get running reports details`()
+    {
+        val instant = Instant.now()
+        val fakeReportRunLog = ReportRunLog(
+                "test@example.com",
+                instant,
+                "q123",
+                mapOf("annex" to "production", "source" to "production"),
+                mapOf("name" to "cologne", "value" to "memo"),
+                "branch",
+                "commit",
+                "complete",
+                "logs",
+                "1233")
+
+        val mockContext: ActionContext = mock {
+            on { params(":key") } doReturn "fakeKey"
+        }
+
+        val mockRepo = mock<ReportRunRepository> {
+            on { getReportRun("fakeKey") } doReturn fakeReportRunLog
+        }
+
+        val sut = ReportRunController(mockContext, mockRepo)
+        val result = sut.getRunningReportLogs()
+        assertThat(result).isEqualTo(fakeReportRunLog)
+    }
+
+    @Test
+    fun `only report runners can get running report logs`()
+    {
+        insertUser("user@email.com", "user.name")
+        val now = Instant.now()
+        val sut = OrderlyWebReportRunRepository()
+        sut.addReportRun(
+                "frightened_rabbit",
+                "user@email.com",
+                now,
+                "report1",
+                mapOf("instance1" to "pre-staging"),
+                mapOf("parameter1" to "value1"),
+                "branch1",
+                "commit1"
+        )
+
+        val url = "/running/frightened_rabbit/logs"
+        val response = webRequestHelper.loginWithMontaguAndMakeRequest(url,
+                setOf(ReifiedPermission("reports.run", Scope.Global())),
+                method = HttpMethod.get,
+                contentType = ContentTypes.json)
+
+        assertSuccessful(response)
+        assertJsonContentType(response)
```

Comment:
Two issues here: you need to check the response itself (it could be empty). Also: this isn't testing that only report runners can get logs - you need to also check that it fails without the `reports.run` permission.